src/components/Workshop/ForgeTabPanel.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import Divider from '@material-ui/core/Divider'
+import ReactMarkdown from 'react-markdown'
+
+import { recipeType } from '../../enums'
+
+import { recipeCategories } from '../../data/maps'
+
+import UpgradePurchase from '../UpgradePurchase'
+
+import { TabPanel } from './TabPanel'
+import { RecipeList } from './RecipeList'
+
+import { getUpgradesAvailable } from './getUpgradesAvailable'
+
+export function ForgeTabPanel({
+  currentTab,
+  learnedForgeRecipes,
+  setCurrentTab,
+  toolLevels,
+}) {
+  const upgradesAvailable = getUpgradesAvailable({
+    toolLevels,
+    learnedForgeRecipes,
+  })
+
+  return (
+    <TabPanel value={currentTab} index={1}>
```

Comment:
`index={1}` seems to assume a lot about how this component will be used. Do you think it would be worth making `index` a required prop of this component so that the consumer is in charge of tab order?

Reply:
oh yeah, great suggestion!